addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -131,8 +131,7 @@ class ThreadView extends Component {
             thread,
             threadCacheIsLoaded: threadCache && threadCache.isLoaded,
             threadIsTemporary: thread && thread.isTemporary,
-            threadMassMailing: thread && thread.mass_mailing,
-            threadModel: thread && thread.model,
+            threadTextInputSendShortcuts: thread && thread.textInputSendShortcuts,
```

Comment:
`textInputSendShortcuts` being an array, you'll need to use `compareDepth` option with it, otherwise it will fail to update the component if the elements are added/removed from the array (this should be a non-issue with current code because the array is always fully replaced in the compute method, but better safe than sorry).